packages/gestalt/src/TableRowExpandable.js

```diff
@@ -58,7 +58,9 @@ export default function TableRowExpandable({
   hoverStyle = 'gray',
 }: Props): Node {
   const [expanded, setExpanded] = useState(false);
-  const cs = hoverStyle === 'gray' ? cx(styles.hoverShadeGray) : null;
+  const cstr = hoverStyle === 'gray' ? cx(styles.hoverShadeGray) : null;
```

Comment:
`classnames` isn't doing anything here — just use the class names directly where needed

`classnames` is only useful when combining multiple classes, especially if there's conditional logic for which classes are included

Reply:
DONE